app/resources/view.html.template

```diff
@@ -17,15 +17,22 @@
   {{person.full_name}} - {{block.super}}
 {% endblock %}
 
-{% if params.query or params.given_name or params.family_name %}
-  {% block logo %}
+{% block logo %}
+  {% if params.query_name or params.given_name or params.family_name %}
```

Comment:
yes,
I found there will have some problems for the original order of the code when user want to create a new record for a person who already has some similar search records. if user create a new record for this person and click go back arrow, it will return to the search result error page instead of the front page. and for the current order, it will return to the front page.

edited else part to {{block.super}}